modal/image.py

```diff
@@ -1299,6 +1299,68 @@ def build_dockerfile(version: ImageBuilderVersion) -> DockerfileSpec:
             gpu_config=parse_gpu_config(gpu),
         )
 
+    def uv_sync(
+        self,
+        lockfile: str = "./uv.lock",  # Path to lockfile
+        # Path to pyproject.toml. If not provided, then use file in the same directory as the lockfile
+        pyproject_toml: Optional[str] = None,
+        *,
+        force_build: bool = False,  # Ignore cached builds, similar to 'docker build --no-cache'
+        secrets: Sequence[_Secret] = [],
+        gpu: GPU_T = None,
+    ) -> "_Image":
+        """Install dependencies from a local `uv.lock` file using `uv sync`.
+
+        Note that only dependencies are installed. Please use `add_local_python_source` to include local python source
```

Comment:
How would people that are fully bought-into the `uv sync` workflow normally be handling "local" dependencies?